package.json

```diff
@@ -8,7 +8,7 @@
     "node": ">=4.0.0"
   },
   "repository": "googleapis/nodejs-logging-winston",
-  "main": "./src/index.js",
+  "main": "./build/src/index.js",
   "files": [
     "src",
```

Comment:
Did you miss the changes to this section?

Reply:
I've added a `types` attribute and updated the `files` section.